### ISSUE TITLE
Import/Export database crash

### ISSUE THREAD

**Issue by ITMacch** [TeamNewPipe/NewPipe#7783]
### Checklist

- [x] I am using the latest version - 0.21.16 
- [x] I checked, but didn't find any duplicates (open OR closed) of this issue in the repo. 
- [x] I have read the contribution guidelines given at https://github.com/TeamNewPipe/NewPipe/blob/HEAD/.github/CONTRIBUTING.md.
- [ ] This issue contains only one bug. I will open one issue for every bug report I want to file.

### Steps to reproduce the bug
1. Go to 'Settings'
2. Press on 'Content'
3. Swipe down to 'Import database' or 'Export database' 

### Actual behavior

App crashes

### Expected behavior

File Browser should open to select/save database

## Exception
* __User Action:__ ui error
* __Request:__ ACRA report
* __Content Country:__ DE
* __Content Language:__ en-DE
* __App Language:__ en_DE
* __Service:__ none
* __Version:__ 0.21.16
* __OS:__ Linux samsung/m51nseea/m51:11/RP1A.200720.012/M515FXXU3CUH2:user/release-keys 11 - 30
<details><summary><b>Crash log </b></summary><p>

```
android.content.ActivityNotFoundException: No Activity found to handle Intent { act=android.intent.action.OPEN_DOCUMENT cat=[android.intent.category.OPENABLE] typ=application/zip flg=0x43 (has extras) }
	at android.app.Instrumentation.checkStartActivityResult(Instrumentation.java:2080)
	at android.app.Instrumentation.execStartActivity(Instrumentation.java:1727)
	at android.app.Activity.startActivityForResult(Activity.java:5377)
	at androidx.activity.ComponentActivity.startActivityForResult(ComponentActivity.java:597)
	at androidx.core.app.ActivityCompat.startActivityForResult(ActivityCompat.java:237)
	at androidx.activity.ComponentActivity$2.onLaunch(ComponentActivity.java:210)
	at androidx.activity.result.ActivityResultRegistry$2.launch(ActivityResultRegistry.java:167)
	at androidx.fragment.app.Fragment$9.launch(Fragment.java:3510)
	at androidx.activity.result.ActivityResultLauncher.launch(ActivityResultLauncher.java:47)
	at org.schabi.newpipe.settings.ContentSettingsFragment.lambda$onCreatePreferences$0(ContentSettingsFragment.java:76)
	at org.schabi.newpipe.settings.ContentSettingsFragment.$r8$lambda$IJ0-sY_2nFZ8j0D13C6AQ28cAn0(Unknown Source:0)
	at org.schabi.newpipe.settings.ContentSettingsFragment$$ExternalSyntheticLambda7.onPreferenceClick(Unknown Source:2)
	at androidx.preference.Preference.performClick(Preference.java:1184)
	at androidx.preference.Preference.performClick(Preference.java:1166)
	at androidx.preference.Preference$1.onClick(Preference.java:181)
	at android.view.View.performClick(View.java:8160)
	at android.view.View.performClickInternal(View.java:8137)
	at android.view.View.access$3700(View.java:888)
	at android.view.View$PerformClick.run(View.java:30236)
	at android.os.Handler.handleCallback(Handler.java:938)
	at android.os.Handler.dispatchMessage(Handler.java:99)
	at android.os.Looper.loop(Looper.java:246)
	at android.app.ActivityThread.main(ActivityThread.java:8544)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:602)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1130)

```
</details>
<hr>

### Device info

 - Android version/Custom ROM version: 11 - OneUI Core 3.1
 - Device model: Samsung Galaxy M51

**Comment by mhmdanas** [comment]
Do you have a file manager installed? if not, try installing one (I suggest the one I linked to below).

If you use a system file manager, I've noticed from previous reports that some of them don't seem to work properly with SAF. Can you try with https://f-droid.org/packages/me.zhanghai.android.files and see if it works?

**Comment by ITMacch** [comment]
I had only the samsung file Manager (my files) and also disabled with package manager 'files' (com.google.android. documentsui). I enabled it and the problem is fixed. Thx and sorry!

**Comment by litetex** [comment]
Was fixed by #7452
